karma_chameleon/bot.py

```diff
@@ -59,18 +59,18 @@ def _load_karma_from_json_file(self) -> None:
         karma_file = Path(self.karma_file_path)
         if not karma_file.is_file():
             self.logger.debug("No existing karma file found. Will start fresh.")
-            with open(self.karma_file_path, "w") as file_ptr:
+            with open(self.karma_file_path, "w", encoding="utf-8") as file_ptr:
```

Comment:
Why do we need to explicitly pass `utf-8` as our encoding here? It looks like by default [it will just use whatever the preferred encoding is on the machine](https://docs.python.org/3/library/locale.html#locale.getpreferredencoding).

Reply:
That was my impression as well-- I was just making the linter happy with this change.